backend/src/bundles/user-details/user-details.repository.ts

```diff
@@ -64,6 +64,18 @@ class UserDetailsRepository implements Repository {
         });
     }
 
+    public async findUnconfirmedByRole(
+        role: 'talent' | 'employer',
+    ): Promise<UserDetailsModel[]> {
+        return await this.userDetailsModel
```

Comment:
Don't use return await

```suggestion
        return this.userDetailsModel
```

Reply:
fixed